pkg/deploy/dev-workspace-config/dev_workspace_config.go

```diff
@@ -131,3 +135,8 @@ func updateWorkspaceServiceAccountConfig(devEnvironments *chev2.CheClusterDevEnv
 	}
 	return nil
 }
+
+func updateWorkspacePodSchedulerNameConfig(devEnvironments *chev2.CheClusterDevEnvironments, workspaceConfig *controllerv1alpha1.WorkspaceConfig) error {
```

Comment:
I just noticed this function doesn't ever return an error. @tolusha  if desired, I can modify it as an extra commit in https://github.com/eclipse-che/che-operator/pull/1576 to remove the unnecessary error handling to simplify things a bit.